src/ansys/dpf/composites/sampling_point.py

```diff
@@ -74,6 +76,7 @@ class SamplingPointNew(SamplingPoint):
     typically three integration points through the thickness. The through-the-thickness
     integration points are called `spots`. They are typically at the ``BOTTOM``, ``MIDDLE``,
     and ``TOP`` of the layer. This notation is used here to identify the corresponding data.
+    Note that solid elements have only two spots (``BOTTOM`` and ``TOP``).
```

Comment:
The `SamplingPointNew` class is not included in the docs. If that's intentional [1], should the `SamplingPoint` docstring be updated to include this information (also the changes below)? 

[1] guess it might, since we otherwise expose the subclass names into the public interface.